docs/UserGuide.md

```diff
@@ -119,14 +119,12 @@ The `Tool Bar` at the top provides a set of configuration options that control t
   * `Author` : results will be grouped by the name of the author. Contributions made to multiple repositories by a particular author will be grouped under the author.
 * `Sort groups by`: sorting criteria for the main group
   * `Group title` : groups will be sorted by the title of the group (in bold text) in alphabetical order.
-  * `Repo/branch` : groups will be sorted in alphabetical order by the name of the repo, followed by name of the branch. See note [1] below.
   * `Contribution` : groups will be sorted by the combined contributions within a group, in the order of number of lines added
   * `Variance` : groups will be sorted by the average of the squared differences from the average number of lines of code contributed per day among all authors involved. Detailed definition of variance is located [here](https://en.wikipedia.org/wiki/Variance).
 * `Sort within groups by`: sorting criteria within each group
-  * `Title` : each sub-group will be sorted by it's title in alphabetical order.
-  * `Repo/branch` : each sub-group will be sorted in alphabetical order by the name of the repo, followed by name of the branch. See note [1] below.
-  * `Contribution` : each sub-group will be sorted by individual contributions in the order of number of lines added
-  * `Variance` : each sub-group will be sorted by the average of the squared differences from the average number of lines of code contributed per day by each author into a particular repo. Detailed definition of variance is located [here](https://en.wikipedia.org/wiki/Variance).
+  * `Title` : each group will be sorted internally by it's title in alphabetical order.
+  * `Contribution` : each group will be sorted internally by individual contributions in the order of number of lines added
+  * `Variance` : each group will be sorted internally by the average of the squared differences from the average number of lines of code contributed per day by each author into a particular repo. Detailed definition of variance is located [here](https://en.wikipedia.org/wiki/Variance).
```

Comment:
maybe can simplify the explanation to in terms of english e.g. `how far the daily contributions are spread out from their average value`